academy/0.0-B9lab-Blockchains/5_crypto.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Cryptography"
 order: 6
-description: Cryptographic Foundations
+description: Cryptographic fundamentals of blockchain - Public-key cryptography
 tag: fast-track
 ---
 
-# Cryptographic fundamentals of blockchain: Public-key cryptography
+# Cryptographic fundamentals of blockchain - Public-key cryptography 
```

Comment:
Usually you repeat the title, not the description

Reply:
Those can be touched in another pass